### ISSUE TITLE
nrf82: reel_board: USB unable to send fragmented packets through Control endpoint

### ISSUE THREAD

**Issue by finikorg** [zephyrproject-rtos/zephyr#11232]
When testing issue with wpanusb I noticed that after Host sends more then 64 bytes through Control endpoint communication stalls.
@mike-scott

**Comment by carlescufi** [comment]
Assigned to @pawelzadrozniak so he can take a look.

**Comment by finikorg** [comment]
![image](https://user-images.githubusercontent.com/3072104/48263332-65105d80-e42e-11e8-8cbe-06d191c6db45.png)

**Comment by pawelzadrozniak** [comment]
I cannot reproduce the problem. CPU hangs in radio IRQ handler after "iwpan phy $PHY set channel 0 26".

**Comment by finikorg** [comment]
> I cannot reproduce the problem. CPU hangs in radio IRQ handler after "iwpan phy $PHY set channel 0 26".

Do you mean Linux hangs? BTW: have you disabled Bluetooth? Looks like reel_board always include BT in every project ;)

**Comment by jfischer-phytec-iot** [comment]
>Do you mean Linux hangs? BTW: have you disabled Bluetooth? Looks like reel_board always include BT in every project ;)

:smile: well, like other NRF based boards:
boards/arm/reel_board/reel_board_defconfig:24:CONFIG_BT_CTLR=y

**Comment by carlescufi** [comment]
@finikorg 
>that always enabling Bluetooth, is this a bug?

Well, we do the same for all nRF-based boards. That said this might not be ideal, but in any case we would need to revisit all the boards, so this is outside the scope of this issue.

**Comment by carlescufi** [comment]
>Is this correct that BT does not work with 802.15.4 and the board crashes when both are enabled?

Yes, this is correct since both access the radio and are not coordinated.
